brushfire-core/src/main/scala/com/stripe/brushfire/Injections.scala

```diff
@@ -88,11 +88,10 @@ object JsonInjections {
             case EqualTo(v) => obj.put("eq", toJsonNode(v))
             case LessThan(v) => obj.put("lt", toJsonNode(v))
             case Not(pred) => obj.put("not", toJsonNode(pred)(predicateJsonNodeInjection))
-            case AnyOf(preds) => {
+            case AnyOf(preds) =>
```

Comment:
Huh, it's really unintuitive to me that you don't need braces around a multi-expression case like that. (Though it certainly seems to be true that you don't).